Userland/Libraries/LibWeb/HTML/Policy.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2024, Noah Bright <noah.bright.1@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/HashTable.h>
+#include <AK/String.h>
+#include <LibWeb/HTML/Origin.h>
+
+// https://w3c.github.io/webappsec-csp/#framework-policy
+namespace Web::HTML {
+
+// https://w3c.github.io/webappsec-csp/#directives
+struct Directive {
+    String key;
+    HashTable<String> value;
+};
+// FIXME: implement directive algorithms
+
+class Policy {
```

Comment:
this should link `https://w3c.github.io/webappsec-csp/#content-security-policy-object`

Reply:
Is there any reason to not just keep this as a POD struct with public members? We use that pattern for a lot of things.